### PR TITLE
77 Change appt status

### LINKED ISSUE #77: Update appointment status
As a user I can update my appointment status.

### PR DESCRIPTION
**Pull request by davidlopez**
## Type of change
- [x] :hatching_chick: New feature (non-breaking change which adds functionality)
- [x] :page_with_curl: This change requires a documentation update

## Summary of changes
- Update README for appointment status endpoint
- Update appointment_detail endpoint to update status

## How Has This Been Tested?
- [x] :black_square_button: Integration testing

## Checklist:
- [x] :white_check_mark: I have performed a self-review of my own code
- [x] :eight_spoked_asterisk: I have commented my code, particularly in hard-to-understand areas
- [x] :page_facing_up: I have made corresponding changes to the documentation
- [x] :no_entry_sign: My changes generate no new warnings
- [x] :heavy_check_mark: I have added tests that prove my fix is effective or that my feature works
- [x] :100: New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Status: 200
 ```
 
 ###### Make an Appointment
-`POST` to `/api/v1/notaries/:user_id/appointments`
+`POST` to `/api/v1/notaries/:notary_user_id/appointments`
```
Comment (lrs8810) [inline README.md:185]: Nice job making this more specific!

File: notary_now_api/tests.py [hunk 2]
```diff
@@ -199,3 +199,85 @@ def test_appointment_endpoint_sad_path(self):
         request = self.factory.get(f'/api/v1/notaries/20/appointments/{self.appointment_one.id}')
         response = appointment_detail(request, 20, self.appointment_one.id)
         self.assertEqual(response.status_code, 400)
+
+class ChangeAppointmentStatusTest(TestCase):
```
Comment (lrs8810) [inline notary_now_api/tests.py:203]: Great job TDDing!

File: notary_now_api/tests.py [hunk 3]
```diff
@@ -199,3 +199,85 @@ def test_appointment_endpoint_sad_path(self):
...
+            "status": "CANCELLED"
+         }),
+         content_type='application/json')
+
+        response = appointment_detail(request, self.user.id, self.appointment_one.id)
+        json_response = json.loads(response.content)
+
+        self.assertEqual(json_response['status'], 'Cancelled')
+
+    def test_appointment_status_endpoint_sadpath(self):
```
Comment (lrs8810) [inline notary_now_api/tests.py:258]: Love the sad paths!

**Review by lrs8810 — APPROVED** [review]
Great job!

**Review by BabsLabs — APPROVED** [review]
This looks great. Good use of the ApplicationStatus enum.

**Pull request merged**